packages/nimble-components/src/anchor-menu-item/styles.ts

```diff
@@ -55,6 +51,13 @@ export const styles = css`
         cursor: pointer;
         text-decoration: none;
         outline: none;
+        padding-left: 8px;
+        padding-right: 8px;
+    }
+
+    a${focusVisible} {
```

Comment:
I was wondering why the old styling removed the ring when the menu item was pressed/active, since the new styling does not. The new styling is consistent with non-anchor menu items, so I'm not sure what the motivation was before. Anyway, seems correct now.